vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/gemini/post_recipient_id/post_recipient_id_gemini.cc

```diff
@@ -95,6 +95,15 @@ void PostRecipientId::OnRequest(const type::UrlResponse& response,
                                 PostRecipientIdCallback callback) {
   ledger::LogUrlResponse(__func__, response);
 
+  auto header = response.headers.find("www-authenticate");
+  if (header != response.headers.end()) {
+    std::string auth_header = response.headers.at("www-authenticate");
```

Comment:
Can this be `std::string& auth_header = header->second;` to avoid the additional lookup and string copy?

Reply:
fixed!